src/Views/display.php

```diff
@@ -1,9 +1,9 @@
 <?php foreach ($alerts as $type => $messages): ?>
     <?php foreach ($messages as $message): ?>
-        <div x-data="{ show: true }" x-show="show" x-init="setTimeout(() => show = false, <?= $message['displayTime']; ?>)" class="toast show mt-1" role="alert" aria-live="assertive" aria-atomic="true">
+        <div x-data="{ show: true }" x-show="show" x-init="setTimeout(() => show = false, <?= esc($message['displayTime']); ?>)" class="toast show mt-1" role="alert" aria-live="assertive" aria-atomic="true">
```

Comment:
We don't have to escape `displayTime` as it's always an integer value.

Reply:
Good point.

Reply:
Thanks!